core/src/main/java/UIElements/TriviaButtonBuilder.java

```diff
@@ -25,34 +24,50 @@
 
```

Comment:
It sounds like this class is inheriting from Game for the sake of inheriting from something.  Just make this a concrete class with no superclass.